src/time_travel/time_machine_clock.py

```diff
@@ -1,11 +1,18 @@
 """A single clock for all patchers."""
 
 
+MIN_START_TIME = 100000.0  # Windows support unix times starting from 86,400
+
+
 class TimeMachineClock(object):
     """Unifing class for clock types."""
     
-    def __init__(self, start_time=0.0, clock_listeners=None):
-        """Initialise a unifing clock."""
+    def __init__(self, start_time=MIN_START_TIME, clock_listeners=None):
```

Comment:
```
def __init__(..., start_time=None, ...):
  ...
  self._time = MIN_STRAT_TIME if start_time is None else start_time
  ...
```

Reply:
Why? Default arguments were invented for a reason..

Reply:
Well, It's a constant, but still a variable, so can be changed?
You put it under _time, but does it copy it by value? not changing time, will change the constant value?

Reply:
The number is immutable, you can't change the const.

Reply:
Cools